### DISCUSSION TITLE
sqlite3 Unable to Open Db File

### DISCUSSION THREAD

**Discussion by baileyjg** [brianschrameck/arlo-cam-api#11] [Q&A] [answered]
I am working on setting up the arlo-cam-api on my raspberry pi 5 with Linux installed as the OS. I chose the docker-compose method. However when running docker-compose up I run into an error. I feel like it has to do with permissions. I have tried messing around with chmod and chown and to no avail I still keep getting this error.
![dberror](https://github.com/user-attachments/assets/69e7aa2f-b416-4c80-9b54-6465d9b00718)

**Comment by brianschrameck** [comment]
Can you post your docker compose config/yml file?

**Reply by baileyjg** [reply]
Yep, here it is. Both config.yaml and arlo.db are in the same 'ArloBaseStation' directory with this compose file.
![image](https://github.com/user-attachments/assets/8cda2180-d10a-4fc8-9857-ca4a294602fc)

**Reply by brianschrameck** [reply]
Is arlo.db a file or folder? Should be a file.

**Reply by baileyjg** [reply]
So, it was a directory when I got that previous 'unable to open' error. I did `touch arlo.db` this time; now it's a file. However, now I am getting a different error.

![image](https://github.com/user-attachments/assets/8040e0a9-009b-4f53-b90c-38019a29bc0c)

**Reply by brianschrameck** [reply, marked as answer]
I think your container still has it mounted as a directory.

Do `docker compose down`
then
`docker compose system prune –-all` 
then
`docker compose up -d`

**Reply by baileyjg** [reply]
Ah, it looks to be running now. Thanks for the help!
